kitsune/products/jinja2/products/includes/topic_macros.html

```diff
@@ -42,11 +42,9 @@ <h3 class="card--title">
         {{ _('Still need help?') }}
       </h3>
       <p class="card--desc">{{ _('We’re here for you. Post a question to our support forums and get answers from our community of experts.') }}</p>
-      <a
-        class="sumo-button primary-button button-lg"
-        href="{{ url('questions.aaq_step1') }}"
-        data-event-category="link click"
-        data-event-action="topic"
+      <a class="sumo-button primary-button button-lg" {% if product %}
+        href="{{ url('questions.aaq_step2', product_key=product_key)}}" {%else %}
```

Comment:
Should be `{% else %}`

nit: might be cleaner to do the if/else above, and `set` a variable which we then use as the value of the href attribute, like:
```
{% if product %}
  {% set aaq_url = ... %}
{% else %}
  {% set aaq_url = ... %}
{% endif %}
<a href="{{ aaq_url }}" ...